### DISCUSSION TITLE
DatePicker: Show all the format in uppercase or lowercase

### DISCUSSION THREAD

**Discussion by alanzeladaVela1** [rsuite/rsuite#4046] [Q&A] [answered]
Hi everyone!

I wanted to ask if there is a possibility to show all the format of the DatePicker as "mm/dd/yyyy" or "MM/DD/YYYY" and not "MM/dd/yyyy". I know that if I write "mm" it will recognize as minutes but that is not what I want.

I also tried with a placeholder but when I press the DatePicker it shows the format. I know that this is so specific, but I wanted to know if there is a possibility to do this.

Thank you!

**Comment by simonguo** [comment]
Currently not supported, may consider similar solutions in the future: https://github.com/rsuite/rsuite/issues/3924

**Reply by simonguo** [reply, marked as answer]
Another solution is to use the `editable={false}` property.
```tsx
<DatePicker editable={false} placeholder="MM/DD/YYYY" format="MM/dd/yyyy" />
```

**Reply by alanzeladaVela1** [reply]
Thank you! I will use editable={false} by now.
